vmdb/app/presenters/menu/section.rb

```diff
@@ -8,6 +8,15 @@ def features
       Array(items).collect(&:feature).compact
     end
 
+    def visible?(userid)
+      store = Vmdb::PermissionStores.instance
+      tab   = Menu::Manager.tab_features_by_id(id)
+      auth  = store.can?(id) && User.current_user.role_allows_any?(:identifiers => tab)
```

Comment:
@martinpovolny I don't understand.  Doesn't the `User.current_user.role_allows_any?(:identifiers => tab)` handle the RBAC features for the custom menu sections and items?  If not, how do we do it currently?